content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md

```diff
@@ -168,7 +183,11 @@ As stated in the command output, the certificate-key gives access to cluster sen
 
 1.  Apply the CNI plugin of your choice:
 
-    [Follow these instructions](/docs/setup/independent/create-cluster-kubeadm/#pod-network) to install
+<<<<<<< HEAD:content/en/docs/setup/production-environment/tools/independent/high-availability.md
```

Comment:
This looks like a merge left in by mistake.

Reply:
Thank you it was a miss. Fixed it now.

> > Here's the list and this has been updated in the redirects page as well.
> 
> thanks.
> given the tools for `/docs/setup/production-environment` are under `tools/`
> should `/docs/setup/learning-environment/minikube/` be `/docs/setup/learning-environment/tools/minikube/` ?

Hmm... good point. for now there is only minikube in learning-environment. As we get more content for kind and others we could add a new tools directory.